app/templates/index.hbs

```diff
@@ -0,0 +1,39 @@
+<div class="container">
+    <div class='main'>
+        <div class='title'>
+            Welcome to the <br> Ciena Developer's Community
+        </div>
+        <div class='logo'>
+            <img src="./images/frost_logo.png">
+        </div>
+        {{#frost-link 'register'
+        priority='secondary'
```

Comment:
I don't know if we have a standard for this, but I like to see these lines indented, e.g.:
```
                {{#frost-link 'login'
                  priority='secondary'
                  size='small'
                }}
                 Admin Login
               {{/frost-link}}